.github/actionlint.yaml

```diff
@@ -1,6 +1,6 @@
 self-hosted-runner:
   # Pending https://github.com/rhysd/actionlint/issues/533
-  labels: ["windows-11-arm"]
+  labels: ["windows-11-arm", "macos-15-intel"]
```

Comment:
This is in fact no longer pending: https://github.com/rhysd/actionlint/pull/542 / https://github.com/rhysd/actionlint/pull/572
```suggestion
  labels: []
```